tests/functional/oauth_webchannel_force_auth.js

```diff
@@ -3,69 +3,51 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
-  'intern',
   'intern!object',
-  'intern/node_modules/dojo/node!xmlhttprequest',
-  'app/bower_components/fxa-js-client/fxa-client',
   'tests/lib/helpers',
   'tests/functional/lib/helpers',
   'tests/functional/lib/webchannel-helpers'
-], function (intern, registerSuite, nodeXMLHttpRequest, FxaClient, TestHelpers,
-  FunctionalHelpers, WebChannelHelpers) {
-  var config = intern.config;
-  var AUTH_SERVER_ROOT = config.fxaAuthRoot;
+], function (registerSuite, TestHelpers, FunctionalHelpers, WebChannelHelpers) {
+  var thenify = FunctionalHelpers.thenify;
+
+  var createUser = FunctionalHelpers.createUser;
+  var fillOutForceAuth = FunctionalHelpers.fillOutForceAuth;
+  var listenForWebChannelMessage = FunctionalHelpers.listenForWebChannelMessage;
+  var openFxaFromRp = thenify(WebChannelHelpers.openFxaFromRp);
+  var testElementExists = FunctionalHelpers.testElementExists;
+  var testIsBrowserNotifiedOfLogin = WebChannelHelpers.testIsBrowserNotifiedOfLogin;
 
   var PASSWORD = 'password';
   var email;
-  var client;
-
-  var testIsBrowserNotifiedOfLogin = WebChannelHelpers.testIsBrowserNotifiedOfLogin;
-  var openFxaFromRp = WebChannelHelpers.openFxaFromRp;
 
   registerSuite({
     name: 'oauth webchannel force_auth',
     beforeEach: function () {
       email = TestHelpers.createEmail();
 
-      client = new FxaClient(AUTH_SERVER_ROOT, {
-        xhr: nodeXMLHttpRequest.XMLHttpRequest
-      });
-
       return FunctionalHelpers.clearBrowserState(this);
     },
 
     'verified': function () {
-      var self = this;
-      return client.signUp(email, PASSWORD, { preVerified: true })
-        .then(function () {
-          return openFxaFromRp(self, 'force_auth', { email: email })
-            .execute(FunctionalHelpers.listenForWebChannelMessage)
-
-            .then(function () {
-              return FunctionalHelpers.fillOutForceAuth(self, PASSWORD);
-            })
-
-            // the page does not transition, loop will close the screen
-            .findByCssSelector('#fxa-force-auth-header')
-            .end()
-
-            .then(testIsBrowserNotifiedOfLogin(self, { shouldCloseTab: true }));
-        });
+      return this.remote
+        .then(createUser(email, PASSWORD, { preVerified: true }))
+        .then(openFxaFromRp(this, 'force_auth', { query: { email: email }}))
+        .execute(listenForWebChannelMessage)
```

Comment:
`listenForWebChannelMessage` is no longer needed, a WebChannel message listener is attached on page load in FunctionalHelpers.openPage